### PR TITLE
feat(ONYX-1012): add tips for taking photos info modal to photo upload step

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [ONYX-1012] 

### Description

<image src="https://github.com/artsy/eigen/assets/36167539/85bde0dd-aada-4f41-ab25-487147ec3725" width="250"/>

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- add tips for taking photos info modal to photo upload step -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1012]: https://artsyproduct.atlassian.net/browse/ONYX-1012?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (add tips for taking photos info modal to photo upload step -daria)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 60566c2605bf72411e41446ea9b936092f4b72db
</p>

File: src/app/Scenes/SellWithArtsy/ArtworkForm/Components/TipsForTakingPhotos.tsx
Comment (MounirDhahri) [inline src/app/Scenes/SellWithArtsy/ArtworkForm/Components/TipsForTakingPhotos.tsx:1]: I would suggest renaming the filter to `SubmitArtworkTipsForTakingPhotos`

**Review by MounirDhahri — CHANGES_REQUESTED** [review]
The component looks good to me - my only concern is that I believe this should be a modal instead of a separate screen as it belongs to the photos screen and it will make it less coupled to the submit artwork flow. We can use an `InfoMadal` as in here even if the design design match 100%.

**Review by MounirDhahri — APPROVED** [review]

File: src/app/Scenes/SellWithArtsy/ArtworkForm/SubmitArtworkForm.tsx [hunk 1]
```diff
@@ -50,7 +51,7 @@ export type SubmitArtworkStackNavigation = {
 }
 
 export const SubmitArtworkForm: React.FC<SubmitArtworkProps> = (props) => {
-  const initialScreen: SubmitArtworkScreen = props.initialStep || "StartFlow"
+  const initialScreen: SubmitArtworkScreen = props.initialStep || "AddPhotos"
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/ArtworkForm/SubmitArtworkForm.tsx:54]: Is this change intentional?

**Review by olerichter00 — APPROVED** [review]
